doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -1,10 +1,10 @@
-![](img/eu_regional_development_fund_horizontal_div_15.png "European Union | European Regional Development Fund | Investing in your future")
-
----
 
+| ![European Union / European Regional Development Fund / Investing in your future](img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
 
 # Security Server Installation Guide
-**X-ROAD 6**
+
+# X-ROAD 6
```

Comment:
No need for double header. Could be "X-ROAD 6 Security Server Installation Guide" instead, or could be left unchanged